pyproject.toml

```diff
@@ -37,6 +37,7 @@ requires = [
     "pydantic_settings",
     "redis>=4.6, <4.7.0",
     "structlog~=22.1.0",
+    "strawberry-graphql==0.214.0",
```

Comment:
This shouldn't be necessary

Reply:
I removed the version so apps that use this library can be more specific if needed.